Firestore/core/src/local/local_write_result.h

```diff
@@ -31,6 +31,17 @@ namespace local {
 /** The result of a write to the local store. */
 class LocalWriteResult {
  public:
+  static LocalWriteResult FromOverlayedDocuments(
+      model::BatchId batch_id, model::OverlayedDocumentMap&& overlayed_docs) {
+    model::DocumentMap docs;
+    for (const auto& overlayed_doc : overlayed_docs) {
+      docs = docs.insert(overlayed_doc.first,
+                         std::move(overlayed_doc.second.document()));
```

Comment:
Remove the `std::move` since it has no effects because OverlayedDocumentMap only supports const iterators.